### ISSUE TITLE
Help and alive command

### ISSUE THREAD

**Issue by double-fault** [Tunaki/burnaki#1]
`help` and `alive` commands need to be implemented for Burnaki. The `alive` command is very important.

**Comment by Tunaki** [comment]
Yeah that makes sense, I'll see about adding those. The most difficult part is coming with a nice pun in the `alive` response :).
